src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTestCase.php

```diff
@@ -91,8 +90,6 @@
 
 abstract class FrameworkExtensionTestCase extends TestCase
 {
-    use ExpectDeprecationTrait;
```

Comment:
This would need to be readied once we deprecate sth. in Symfony 7, right?

Reply:
indeed